node/lib/task.ts

```diff
@@ -119,27 +119,55 @@ var locStringCache: {
 } = {};
 var resourceFile: string;
 var libResourceFileLoaded: boolean = false;
+var resourceCulture: string = 'en-US';
+
+function loadResJson(resjsonFile: string): any {
+    var resJson: {} = null;
+    if (exist(resjsonFile)) {
+        var resjsonContent = fs.readFileSync(resjsonFile, 'utf8').toString();
+        // remove BOM
+        if (resjsonContent.indexOf('\uFEFF') == 0) {
+            resjsonContent = resjsonContent.slice(1);
+        }
+
+        try {
+            resJson = JSON.parse(resjsonContent);
+        }
+        catch (err) {
+            debug('unable to parse resjson with err: ' + err.message);
+            resJson = null;
+        }
+    }
+    else {
+        debug('.resjson file not found: ' + resjsonFile);
+    }
 
-function loadLocStrings(resourceFile: string): { [key: string]: string; } {
+    return resJson;
+}
+
+function loadLocStrings(resourceFile: string, culture: string): { [key: string]: string; } {
     var locStrings: {
         [key: string]: string
     } = {};
 
     if (exist(resourceFile)) {
-        debug('load loc strings from: ' + resourceFile);
+        debug('load strings from: ' + resourceFile);
         var resourceJson = require(resourceFile);
 
         if (resourceJson && resourceJson.hasOwnProperty('messages')) {
+            var locResourceJson = null;
+            // load up resource resjson for different culture
+            var localizedResourceFile = path.join(path.dirname(resourceFile), 'strings', 'resources.resjson', culture, 'resources.resjson');
+            if (exist(localizedResourceFile)) {
```

Comment:
Not a big deal, but I noticed that you check `if exist` here and inside the function.